setup.cfg

```diff
@@ -76,7 +71,7 @@ test =
 where = src
 
 [options.package_data]
-ansiblelint = py.typed
+ansiblecompat = py.typed
```

Comment:
unrelated question, but why the importable name is "ansiblecompat" and not "ansible_compat"?

Reply:
That is because I followed naming used by `ansiblelint` package. In fact I personally prefer easier to read imports, but at the same time I do not enjoy having some with and some without separator. Do we already have some other packages starting with ansible to compare against?

Reply:
Does https://github.com/ansible/ansible-navigator/tree/main/ansible_navigator count?

Reply:
Also https://github.com/ansible/ansible-builder/tree/devel/ansible_builder

Reply:
And finally https://github.com/ansible-community/ansible-bender/tree/master/ansible_bender (off the top of my head)

Reply:
Maybe there's others but this should be enough to state that there are projects that follow both practices. But from a Python dev PoV (readability and style guides), I'd prefer an underscore. It's probably best to do this in a separate PR, though.

Reply:
I will fix that in follow up, 3 examples was more than enough. I wonder if we do the same for linter in the future.